src/neo/Network/P2P/Payloads/CosignerAttribute.cs

```diff
@@ -0,0 +1,40 @@
+using Neo.IO;
+using Neo.IO.Json;
+using System;
+using System.IO;
+using System.Linq;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class CosignerAttribute : TransactionAttribute
+    {
+        public override int Size => Cosigners.GetVarSize();
+        public override TransactionAttributeUsage Usage => TransactionAttributeUsage.Cosigners;
+
+        public Cosigner[] Cosigners { get; set; }
```

Comment:
Why not one cosigner per attribute?

Reply:
I think that it will be faster if we only allow one attribute per kind, otherwise it's possible to attach multiple oracle Request. what do you think?
https://github.com/neo-project/neo/pull/1620#discussion_r417781903

Reply:
I think multiple request is okay. We need to allow multiple request.